### DISCUSSION TITLE
Unable to Purchase or transfer aircraft to captured airbase or FOB

### DISCUSSION THREAD

**Discussion by cazubris** [Khopa/dcs_liberation#2181] [Q&A] [answered]
I am playing through one of the Syria campaigns and I've captured 2 FOBs and one airbase. I'm unable to purchase new aircraft, only ground vehicles. I can't seem to transfer aircraft to these locations either. I have even tried to make the divert and return location one of these captured locations. I can't seem to get my Helo's to divert and I can't figure out how to make them land at a specific location using the waypoint editor. 
Does anyone have an idea for this? As the campaign goes on the helo ingress time gets worse and worse. Need to be able to move units up closer to the front line fobs.

**Comment by Mustang-25** [comment]
Does the FOB have helo slots? Depending on the campaign/campaign creator, not every FOB can support helos.

**Comment by Starfire13** [comment]
Please read the wiki, specifically the section on unit transfers so you know how to move squadrons.

**Comment by X-Quetzalcoatl-X** [comment]
I also have this issue. SHouldn't a captured airbase with a factory be able to produce units locally?

**Comment by Starfire13** [comment]
> I also have this issue. SHouldn't a captured airbase with a factory be able to produce units locally?

Are you having troubles with air or ground units?

**Comment by X-Quetzalcoatl-X** [comment]
AIr units
I can use the ferry method to relocate air squadrons np.

**Reply by Starfire13** [reply, marked as answer]
> AIr units I can use the ferry method to relocate air squadrons np.

Factories are for obtaining ground units, not air units. If you transfer a squadron to an airbase, you can buy aircraft there on the next turn. There is a page explaining unit transfers on the wiki.
